include/ustore/cpp/docs_ref.hpp

```diff
@@ -68,7 +68,7 @@ class docs_ref_gt {
   protected:
     ustore_database_t db_ = nullptr;
     ustore_transaction_t transaction_ = nullptr;
-    ustore_snapshot_t snapshot_ = nullptr;
+    ustore_snapshot_t snapshot_ = {};
```

Comment:
We either initialize with `{}` or with `= ...`, but not with both, especially when every other member is initialized differently.